tests/core/templates/test_command_queue.h

```diff
@@ -244,8 +244,49 @@ class SharedThreadState {
 		}
 		writer_thread.wait_to_finish();
 	}
+
+	struct CopyMoveTestType {
```

Comment:
Even if this is not regular code, I'd still fancy some `p_` prefix in the parameter names throughout it.

Reply:
Fair enough! I will fix that.